testing/handlertest/testhandler.go

```diff
@@ -0,0 +1,181 @@
+// Package handlertest implements support for testing implementations
+// of HTTP handlers. This is inspired by fstest.TestFS.
+package handlertest
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/http-wasm/http-wasm-host-go/api/handler"
+)
+
+// TestHost tests a handler.Host by checking default property values and
+// ability to change them.
+//
+// To use this, pass your host and the want protocol version.
```

Comment:
Think the protocol version may be drift it takes `newCtx`

Reply:
yep that was drift!